docs/topics/db/queries.txt

```diff
@@ -892,14 +893,21 @@ array::
 If the key you wish to query by clashes with the name of another lookup, use
 the :lookup:`contains <jsonfield.contains>` lookup instead.
 
+To query for missing keys, use the ``isnull`` lookup::
+
+    >>> Dog.objects.create(name='Shep', data={'breed': 'collie'})
+    <Dog: Shep>
+    >>> Dog.objects.filter(data__owner__isnull=True)
+    <QuerySet [<Dog: Shep>]>
+
 .. note::
 
-    The transform examples given above implicitly use the :lookup:`exact`
-    lookup. Key, index, and path transforms can also be chained with
-    :lookup:`iexact`, :lookup:`contains`, :lookup:`icontains`,
-    :lookup:`startswith`, :lookup:`istartswith`, :lookup:`endswith`,
-    :lookup:`iendswith`, :lookup:`regex`, :lookup:`iregex`, :lookup:`lt`,
-    :lookup:`lte`, :lookup:`gt`, and :lookup:`gte` lookups.
+    The lookup examples given above implicitly use the :lookup:`exact` lookup.
+    Key, index, and path transforms can also be chained with:
+    :lookup:`contains`, :lookup:`icontains`, :lookup:`endswith`,
+    :lookup:`iendswith`, :lookup:`iexact`, :lookup:`regex`, :lookup:`iregex`,
+    :lookup:`startswith`, :lookup:`istartswith`, :lookup:`lt`, :lookup:`lte`,
+    :lookup:`gt`, and :lookup:`gte` lookups.
```

Comment:
Do we need to explicitly state that we can also chain them with `JSONField` lookups (e.g. `has_key`, `has_any_keys`, except `contains`)?

Reply:
:thinking: How we can chain `has_key` with these lookup (e.g. `endswith`)?

Reply:
No, I mean `Key, index, and path transforms` can be chained with `JSONField`-only lookups.

Reply:
I don't think we need to document this. Personally, I'm happy with current docs changes.